lib/workers/global/autodiscover.ts

```diff
@@ -38,6 +38,8 @@ export async function autodiscoverRepositories(
   // Autodiscover list of repositories
   let discovered = await platform.getRepos({
     topics: config.autodiscoverTopics,
+    sort: config.autodiscoverRepoSort,
+    order: config.autodiscoverRepoOrder,
```

Comment:
Won't this pass through `null` by default?

Reply:
Yes. But this should be safe as we confirm that the value is not null/undefined before passing them as request params in the `fetchRepositories` call.

Reply:
It's still incorrect typings which I prefer to avoid